client/payment-details/timeline/test/captured-event-strings.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * External dependencies
+ */
+import path from 'path';
+import fs from 'fs';
+
+/**
+ * Internal dependencies
+ */
+import {
+	composeFXString,
+	composeFeeString,
+	feeBreakdown,
+	composeNetString,
+} from '../map-events';
+
+describe( 'Strings in captured events', () => {
+	beforeEach( () => {
+		jest.clearAllMocks();
+		global.wcpaySettings = {
+			zeroDecimalCurrencies: [ 'vnd' ],
+			connect: {
+				country: 'US',
+			},
+			currencyData: {
+				US: {
+					code: 'USD',
+					symbol: '$',
+					symbolPosition: 'left',
+					thousandSeparator: ',',
+					decimalSeparator: '.',
+					precision: 2,
+				},
+				FR: {
+					code: 'EUR',
+					symbol: '€',
+					symbolPosition: 'right_space',
+					thousandSeparator: ' ',
+					decimalSeparator: ',',
+					precision: 2,
+				},
+				GB: {
+					code: 'GBP',
+					symbol: '£',
+					symbolPosition: 'left',
+					thousandSeparator: ',',
+					decimalSeparator: '.',
+					precision: 2,
+				},
+			},
+		};
+	} );
+
+	const loadFixtures = () => {
+		const dir = path.join(
+			__dirname,
+			'../../../../',
```

Comment:
Is this an ideal way to navigate?

Reply:
Yeah, it's not ideal but I am not aware of anyway of retrieving the `rootDir` value from [the Jest config file](https://github.com/Automattic/woocommerce-payments/blob/a7d5f01fb8e60b118a5377855b17d4d699582e1e/tests/js/jest.config.js#L4-L4).

I will keep it as-is now. If anyone has ideas to improve this, feel free to let me know.